text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+experience a similar level of uncertainty around which versions of the compiler
+their library should be compatible with.
+
+We solve this problem by adding long-term support ("LTS") release channels,
+providing explicit support for releases beyond the latest stable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Two new channels: lts and lts-prior
+
+In addition to `stable`, `beta`, and `nightly`, we will add two additional
+release tracs:
+
+* `lts`: The most recent LTS release.
+* `lts-prior`: The second most recent LTS release.
+
+Every fourth stable/beta release, we will perform a release on both the `lts`
+and `lts-prior` channels. The **previous** stable release will be the basis for
+the new `lts` release, and the previous `lts` release will move to the
+`lts-prior` release. This is analogous to the way that the previous beta
+becomes the new stable.
+
+To keep it as easy as possible to remember which releases are lts release, we
+will begin the lts schedule so that the point version of the LTS release is
+always divisible by four. Depending on implementation schedule, that means that
+the first LTS release will probably be 1.28 or 1.32, release concurrently with
+either 1.29 or 1.33.
+
+## Level of support from the Rust project
+
+All three of the stable release channels - stable, lts, and lts-prior - will
+receive the same level of support. We will host documentation for all three
+channels and provide equivalent levels of patch release support - that is, if a
+bug justifying a patch exists in any of these channels, we will release a patch
+for that channel, just as we do today for stable. However, lts releases are
+less likely to receive patch releases in practice, since these bugs are most
+often found in the first six weeks after a stable release.
+
+As a result of this, a stable release which is a candidate for the lts channel
+(that is, its minor version is a multiple of four) will be supported across all
+three channels for the following nine release cycles, a period of 54 weeks or
+just over a year.
+
+## Recommendations for library authors
+
+Today, library authors fall into three general camps:
+
+1. Their library compiles on the nightly channel.
+2. Their library compiles on the stable channel.
+3. Their library compiles on any release more recent than an arbitrary old
+release, which is different for each library (for example, `failure` compiles
+on 1.18.0, whereas `serde` compiles on 1.13.0).
+
+Frequently, whichever level of support they offer, they guarantee it by testing
+their code on that channel or release using CI.
+
+The Rust project recommends that library authors interested in providing
+support for users on versions older than the latest stable, instead of focusing
+on supporting particular *versions*, focus on supporting particular *release
+channels*. That is, a library may guarantee that it will compile on any of the
```

Comment:
I think that doesn't work well and will lead to surprises for users.

Say that you start with adding `lts` to your Travis CI matrix... After half a year, `lts` will suddenly become a different version, and so you are testing a different version, instead of the one you were testing when you added `lts` into your matrix. Therefore, `instead` should become `in addition to`. Otherwise, a crate like `serde` will actually support *fewer* versions (provided `--no-default-features` per @sgrif's note).

Another way to solve this is to always increment the `X` in `lts-X` such that you can pin to a particular lts version. `lts` and `lts-prior` itself will refer to the highest `X` and `X - 1`. Backports will still only be provided for `lts` and `lts-prior`.